### PR TITLE
windows mpi build with impi

### PR DESCRIPTION
**Pull request by minrk**
draft because who knows if this will work

### CODE REVIEW

**Comment by minrk** [comment]
e0dd8b38c59b15b0b9149579fae681b324c16b44 completed successfully, going to try to do some cleanup now.

**Comment by minrk** [comment]
Somewhat surprisingly, seems to work!

**Review by ax3l — APPROVED** [review]
Cool!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -74,13 +76,11 @@ requirements:
     # Keep both versions of openssl in sync
     - openssl
   host:
-    - {{ mpi }}  # [mpi != 'nompi']
+    - {{ mpi | replace("impi", "impi-devel") }}  # [mpi != 'nompi']
```
Comment (hmaarrfk) [inline recipe/meta.yaml:79]: string parsing is "clever" and assumes that `impi` doesn't appear as a substring in other options.

```suggestion
    - {{ mpi }}   # [mpi != 'nompi' or mpi != 'impi']
    - impi-devel  # [mpi == 'impi']
```

**Review by hmaarrfk — APPROVED** [review]

**Pull request merged**